### PR TITLE
Add user support to deleted items for beta

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/directory_deleteditems_get.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 Retrieve the properties of a recently deleted item in [deleted items](../resources/directory.md).
 
-Currently, deleted items functionality is only supported for the [group](../resources/group.md) resource.
+Currently, deleted items functionality is only supported for the [group](../resources/group.md) and [user](../resources/user.md) resources.
 
 ### Prerequisites
-One of the following **scopes** are required to execute this API: *Group.Read.All; Directory.Read.All*
+One of the following **scopes** is required to execute this API: 
+* For users: *User.Read.All; Directory.Read.All*
```
Comment (dkershaw10) [inline api-reference/beta/api/directory_deleteditems_get.md]: You could add Directory.AccessAsUser.All to both of these too.  Same for the list topic.

File: api-reference/beta/api/directory_deleteditems_list.md [hunk 2]
```diff
@@ -2,18 +2,21 @@
 
 Retrieve a list of recently deleted items from [deleted items](../resources/directory.md).
```
Comment (dkershaw10) [inline api-reference/beta/api/directory_deleteditems_list.md:3]: Do you want to add a second example for listing deleted users? NOTE this is a P3 :)

File: api-reference/beta/api/directory_deleteditems_get.md (hunk 1 above)
Comment (maheedharpamuru) [reply api-reference/beta/api/directory_deleteditems_get.md]: Should we only add the minimum required scopes here?   User.ReadWrite.All Group.ReadWrite.All also works right?

**Review by maheedharpamuru — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**